src/docker-outside-of-docker/README.md

```diff
@@ -19,7 +19,7 @@ Re-use the host docker socket, adding the Docker CLI to a container. Feature inv
 |-----|-----|-----|-----|
 | version | Select or enter a Docker/Moby CLI version. (Availability can vary by OS version.) | string | latest |
 | moby | Install OSS Moby build instead of Docker CE | boolean | true |
-| dockerDashComposeVersion | Compose version to use for docker-compose (v1 or v2) | string | v2 |
+| dockerDashComposeVersion | Compose version to use for docker-compose (v1 or v2 or none) | string | v2 |
```

Comment:
Can you address all the comments added for the `docker-in-docker` Feature for `docker-outside-of-docker` Feature as well?